include/bluetooth/gatt.h

```diff
@@ -864,6 +864,17 @@ struct bt_gatt_notify_params {
 int bt_gatt_notify_cb(struct bt_conn *conn,
```

Comment:
I was wondering about that at some point, but I guess the application can take the decision to group or not, besides Id like to avoid extra lockups with notification since that could be time sensitive if the user wants to notify as fast as possible.